tests/test_basic/test_basic.py

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env python
 
+import os
 import sys
-sys.path.insert(0, '..')
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, os.pardir))
```

Comment:
Looks like testing_harness already adds `../..` though.

Sometimes I run individual tests directly, so I think there's value in having it working without run_tests.py setting PYTHONPATH.